charmcraft/commands/store/charmlibs.py

```diff
@@ -244,6 +270,15 @@ def get_libs_from_tree(charm_name=None):
                 for libfile in sorted(v_dir.glob("*.py")):
                     local_libs_data.append(get_lib_info(lib_path=libfile))
 
-    found_libs = [lib_data.full_name for lib_data in local_libs_data]
-    emit.debug(f"Libraries found under {str(base_dir)!r}: {found_libs}")
+    os.chdir(current_directory)
```

Comment:
might want to put this in a `finally:` block (not sure if the functions called before this can raise)

Reply:
No, it should be a `print`, as `charm_builder.py` is run as any other script and its stdout collected (see the `prints` in the rest of the file too)

Reply:
good idea